code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -103,7 +102,8 @@ standOffDist = uq (constrained' (uc sD (variable "SD") Real metre)
     sfwrRange $ Bounded (Inc, sy sdMin) (Inc, sy sdMax)] (exactDbl 45)) defaultUncrt
 
 nomThick = cuc' "nomThick"
-  (nounPhraseSent $ S "nominal thickness t is in" +:+ eS (mkSet Rational (map dbl nominalThicknesses)))
+  (cn "nominal thickness")
+  -- (nounPhraseSent $ S "nominal thickness t is in" +:+ eS (mkSet Rational (map dbl nominalThicknesses)))
```

Comment:
This is related to https://github.com/JacquesCarette/Drasil/issues/4084 . 

I'm in favour of removing this, but I think our policy is to have a replacement in place before we do. Without this here, it won't appear anywhere else in the SRS, which is why we originally left it as-is (#3878).

Reply:
Good point. This is the wrong place, but the information should not completely disappear.

Reply:
My plan is to fix that first. And quite likely, I will end up dropping this particular PR its current form, for a different way to get the same thing done.